examples/nlp_example.py

```diff
@@ -64,6 +63,8 @@ def training_function(config, args):
     datasets = load_dataset("glue", "mrpc")
     metric = load_metric("glue", "mrpc")
 
+    set_seed(seed)
+
```

Comment:
Just as a note, I noticed we set_seed *before* building the dataloaders, hence why this change.